generators/client/templates/angular/src/main/webapp/app/core/login/login.service.ts.ejs

```diff
@@ -99,13 +99,16 @@ export class LoginService {
         this.authServerProvider.logout().subscribe(response => {
             const data = response.body;
             let logoutUrl = data.logoutUrl;
+            // Add here, the custom path if traefik is used
```

Comment:
I think this can be handled in code to get correct redirect uri with base href included like:
```
`${location.origin}${this.location.prepareExternalUrl('/')}`
```
Can you please test and confirm?

Reply:
I confirm that it works. I would update the PR as it is simpler.

Reply:
it's update ;)